packages/health/android/src/main/kotlin/cachet/plugins/health/HealthPlugin.kt

```diff
@@ -457,7 +457,7 @@ class HealthPlugin() : MethodCallHandler, ActivityResultListener, Result, Activi
         for (typeKey in types) {
             if (typeKey !is String) continue
             typesBuilder.addDataType(keyToHealthDataType(typeKey), FitnessOptions.ACCESS_READ)
-            typesBuilder.addDataType(keyToHealthDataType(typeKey), FitnessOptions.ACCESS_WRITE)
+            //typesBuilder.addDataType(keyToHealthDataType(typeKey), FitnessOptions.ACCESS_WRITE)
```

Comment:
Typo? Are you commenting out the write permission on purpose?

Reply:
In our project, only the fitness.*.read scope has applied for permission to access sensitive data and to remove the limit on the number of OAuth users.

Therefore, if we enable ACCESS_WRITE, it will be blocked by Google, so we comment it out to use with our app.

In general, I don't think there is a problem with enabling it, but it would be better if we could set whether to use the write function or not when we new HealthFactory.

Translated with www.DeepL.com/Translator (free version)

Reply:
I'm afraid commenting out the line will have implication on all other users who need the write feature.
But I agree with you that there should be a possibility to opt out the write feature.

By the way, thanks for your quick response :-)

Reply:
> I'm afraid commenting out the line will have implication on all other users who need the write feature.

Yes, I understand that, too. 

I should have made the changes locally instead of pushing them to the repository and I did it.

> But I agree with you that there should be a possibility to opt out the write feature.

I hope you make it.

Thanks,

Reply:
Thanks for your latest commit. 

And I've  filed an enhancement issue #464